GUI/CalibrateTaskWindow.xaml

```diff
@@ -71,34 +87,88 @@
                                 <StackPanel>
                                     <StackPanel Orientation="Horizontal" Margin="5">
                                         <Label x:Name="maxMissedCleavageLabel" Content="Max Missed Cleavages" />
-                                        <TextBox x:Name="missedCleavagesTextBox" PreviewTextInput="PreviewIfInt" Width="45" />
+                                        <TextBox x:Name="missedCleavagesTextBox" PreviewTextInput="PreviewIfInt" Width="45" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                            <TextBox.ToolTip>
+                                                <TextBlock>
+                                                    Maximum number of missed proteolytic cleavages allowed.
+                                                </TextBlock>
+                                            </TextBox.ToolTip>
+                                        </TextBox>
                                     </StackPanel>
                                     <StackPanel Orientation="Horizontal" Margin="5">
                                         <Label x:Name="proteaseLabel" Content="Protease" />
-                                        <ComboBox x:Name="proteaseComboBox" />
+                                        <ComboBox x:Name="proteaseComboBox" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                            <ComboBox.ToolTip>
+                                                <TextBlock>
+                                                        Protease used for digestion.
+                                                        <LineBreak/>
+                                                        Novel proteases can be added/modified by the user at "...MetaMorpheus\\EngineLayer\\Data\\proteases.tsv"
+                                                </TextBlock>
+                                            </ComboBox.ToolTip>
+                                        </ComboBox>
                                     </StackPanel>
                                     <StackPanel Orientation="Horizontal" Margin="5">
                                         <Label x:Name="lblMinPeptideLength" Content="Min Peptide Len" />
-                                        <TextBox x:Name="txtMinPeptideLength" PreviewTextInput="PreviewIfInt" Width="45" />
+                                        <TextBox x:Name="txtMinPeptideLength" PreviewTextInput="PreviewIfInt" Width="45" ToolTipService.ShowDuration="999999" ToolTipService.InitialShowDelay="1000">
+                                            <TextBox.ToolTip>
+                                                <TextBlock>
+                                                        The minimum allowed peptide length. Must be a positive, non-zero integer. Modifications are not counted.
```

Comment:
I don't think we need the "Modifications not counted." Don't people just assume you're counting amino acids only? seems like unnecessary clarification here and elsewhere.